index.js

```diff
@@ -90,7 +100,15 @@ class Conf {
 		// Ensure the directory exists as it could have been deleted in the meantime
 		makeDir.sync(path.dirname(this.path));
 
-		fs.writeFileSync(this.path, JSON.stringify(val, null, '\t'));
+		let data = JSON.stringify(val, null, '\t');
+
+		if (this.encryptionKey) {
+			const cipher = crypto.createCipher('aes-256-cbc', this.encryptionKey);
+			data = cipher.update(data, 'utf8', 'hex');
+			data += cipher.final('hex');
```

Comment:
Wouldn't it be better to write it as a buffer? So without the `'hex'` argument. It would be faster to both read and write. You would then have to conditionally read as a buffer instead of `utf8` too though.

Reply:
I was trying to mimic the current functionality as much as possible - just modify the `data` variable but keep the same type.